genesyscloud/flow_loglevel/genesyscloud_flow_loglevel_proxy.go

```diff
@@ -0,0 +1,160 @@
+}
+
+// createFlowLogLevel creates a Genesys Cloud Flow Log Level
+func (p *flowLogLevelProxy) createFlowLogLevel(ctx context.Context, flowLogLevelId string, flowLogLevelRequest *platformclientv2.Flowloglevelrequest) (*platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	return p.createFlowLogLevelAttr(ctx, p, flowLogLevelId, flowLogLevelRequest)
+}
+
+// getFlowLogLevelById returns a single Genesys Cloud Flow Log Level by Id
+func (p *flowLogLevelProxy) getFlowLogLevelById(ctx context.Context, flowId string) (*platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	return p.getFlowLogLevelByIdAttr(ctx, p, flowId)
+}
+
+// updateFlowLogLevel updates a Genesys Cloud Flow Log Level
+func (p *flowLogLevelProxy) updateFlowLogLevel(ctx context.Context, flowId string, flowLogLevelRequest *platformclientv2.Flowloglevelrequest) (*platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	return p.updateFlowLogLevelAttr(ctx, p, flowId, flowLogLevelRequest)
+}
+
+// DeleteFlowLogLevel deletes a Genesys Cloud Flow Log Level by Id
+func (p *flowLogLevelProxy) deleteFlowLogLevelById(ctx context.Context, flowId string) (*platformclientv2.APIResponse, error) {
+	return p.deleteFlowLogLevelByIdAttr(ctx, p, flowId)
+}
+
+// createFlowLogLevelFn is an implementation function for creating a Genesys Cloud Flow Log Level
+func createFlowLogLevelFn(ctx context.Context, p *flowLogLevelProxy, flowId string, flowLogLevelRequest *platformclientv2.Flowloglevelrequest) (*platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	flowLogLevel, apiResponse, err := p.architectApi.PostFlowInstancesSettingsLoglevels(flowId, *flowLogLevelRequest, nil)
+
+	if err != nil {
+		return nil, apiResponse, fmt.Errorf("Failed to create flow log level: %s", err)
+	}
+
+	return flowLogLevel, apiResponse, nil
+}
+
+// getFlowLogLevelByIdFn is an implementation of the function to get a Genesys Cloud Flow Log Level by Id
+func getFlowLogLevelByIdFn(ctx context.Context, p *flowLogLevelProxy, flowId string) (*platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	flowLogLevel, apiResponse, err := p.architectApi.GetFlowInstancesSettingsLoglevels(flowId, nil)
+	if err != nil {
+		return nil, apiResponse, fmt.Errorf("Failed to retrieve flow log level by id %s: %s", flowId, err)
+	}
+
+	return flowLogLevel, apiResponse, nil
+}
+
+// getAllFlowLogLevelsFn is the implementation for retrieving all flow log levels in Genesys Cloud
+func getAllFlowLogLevelsFn(ctx context.Context, p *flowLogLevelProxy) (*[]platformclientv2.Flowsettingsresponse, *platformclientv2.APIResponse, error) {
+	const pageSize = 100
+	var totalFlowLogLevels []platformclientv2.Flowsettingsresponse
+
+	flowSettingsResponse, apiResponse, err := p.architectApi.GetFlowsInstancesSettingsLoglevels(nil, 1, pageSize)
+	if err != nil {
+		return nil, apiResponse, fmt.Errorf("Failed to get page of flows: %v", err)
+	}
+
```

Comment:
You're just missing one check here, you need to check if `flowSettingsResponse` is nil or if the length of the `Entities` is 0, there should be example in other resource. You'll also need this check in the loop below